bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -488,10 +500,22 @@ public void onValueReceived(String variable, String value, String service) {
                     updateChannel(SPEECHENHANCEMENT);
                     break;
                 case "LineInConnected":
```

Comment:
You may want to introduce const for this ("LineInConnected") and "TOSLinkConnected". They are both used multiple times.

Reply:
Like many others. 
I could add 2 constants for these ones but it exists many other strings in this class without constants.

Reply:
Yes, thanks, appreciated. Just a guess for these two because you touched them in this PR.